packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -343,6 +352,36 @@ The following table lists the set of triggers available, and their corresponding
 For more information on the function of these triggers and how to configure them, read [User Pool Workflows with
 Triggers](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools-working-with-aws-lambda-triggers.html).
 
+You may need to provide your Lambda trigger with permissions to perform operations against your user pool.
```

Comment:
Create a subsection here called 'Trigger permissions' and starting off with a sentence on how to provide permission to lambda trigger. Then follow it up with a paragraph about the lambda trigger needing user pool permissions.

Reply:
took a run at it!